maintainers/maintainer-list.nix

```
@@ -18227,7 +18239,12 @@
     email = "elisp.vim@gmail.com";
     github = "vyp";
     githubId = 3889405;
-    name = "vyp";
+    name = "vyp";nyanbinary = {
+    email = "vextium@skiff.com";
+    github = "nyabinary";
+    githubId =     97130632;
+    name = "Niko";
+}
```

```suggestion
    name = "vyp";
```

Already done on top.